infra/modular-examples/aws-google-workspace/main.tf

```diff
@@ -325,40 +323,38 @@ module "psoxy-bulk-to-worklytics" {
   }, try(each.value.settings_to_provide, {}))
 }
 
-module "psoxy_lookup_tables_builders" {
+# BEGIN lookup builders
+module "lookup_output" {
   for_each = var.lookup_table_builders
 
-  source = "../../modules/aws-psoxy-bulk-existing"
-  # source = "git::https://github.com/worklytics/psoxy//infra/modules/aws-psoxy-bulk-existing?ref=v0.4.13"
-
-  input_bucket                    = module.psoxy-bulk[each.value.input_connector_id].input_bucket
-  aws_account_id                  = var.aws_account_id
-  instance_id                     = each.key
-  aws_region                      = var.aws_region
-  path_to_function_zip            = module.psoxy-aws.path_to_deployment_jar
-  function_zip_hash               = module.psoxy-aws.deployment_package_hash
-  psoxy_base_dir                  = var.psoxy_base_dir
-  rules                           = each.value.rules
-  global_parameter_arns           = module.global_secrets.secret_arns
-  path_to_instance_ssm_parameters = "${var.aws_ssm_param_root_path}PSOXY_${upper(replace(each.key, "-", "_"))}_"
-  ssm_kms_key_ids                 = local.ssm_key_ids
+  source = "../../modules/aws-psoxy-output-bucket"
 
+  instance_id                   = each.key
+  iam_role_for_lambda_name      = module.psoxy-bulk[each.value.input_connector_id].instance_role_name
   sanitized_accessor_role_names = each.value.sanitized_accessor_role_names
-  environment_variables = merge(
-    var.general_environment_variables,
-    try(each.value.environment_variables, {}),
-    {
-      IS_DEVELOPMENT_MODE = contains(var.non_production_connectors, each.key)
-    }
-  )
 }
 
+locals {
+  inputs_to_build_lookups_for = toset(distinct([ for k, v in var.lookup_table_builders : v.input_connector_id ]))
+}
+
+resource "aws_ssm_parameter" "additional_transforms" {
```

Comment:
setting this SSM parameter for each "bulk" lambda to which we want to build an extra lookup table for - eg, but default lambda doesn't have any `ADDITIONAL_TRANSFORMS`; but if you provide this SSM param, then you change the behavior of existing lambda